lighthouse-core/test/lib/minification-estimator-test.js

```diff
@@ -5,9 +5,15 @@
  */
 'use strict';
 
+const fs = require('fs');
+const path = require('path');
 const assert = require('assert');
 const {computeCSSTokenLength, computeJSTokenLength} = require('../../lib/minification-estimator');
 
+
+const nodeModulesPath = path.join(__dirname, '../../../node_modules');
```

Comment:
`require.resolve` can handle all sorts of funky path issues this'll hit.

```js
const angularJSPath = path.join(require.resolve('angular'), 'angular.js');
const angularFullScript = fs.readFileSync(angularJSPath, 'utf8');
```

Reply:
good call. I'll do ya one better, `require.resolve('angular/angular.js')` ;)